app/views/users/_user_info.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="details">
+  <div class="profile">
+    <%= image_tag 'image_profile.jpg', alt: "#{person.name} image" %>
+  </div>
+  <div class="profile-text">
+    <p class="profile-name"> <%= person.name %> </p>
+    <span class="post-num">Number of posts: <%= person.posts.size %> </span>
+  </div>
+</div>
```

Comment:
@Fombi-Favour Great work using partials to render the user, post, and comment details, I loved how it makes it easier to read your code and will make it easier to scale your project in the future.

Reply:
Thank you @danmainah. I really appreciate your feedback